docs/Configuration.md

```diff
@@ -1,6 +1,17 @@
 # Configuration
 
-The GUI can be configured by editing the file `assets/app.config.json`.
+The GUI can be configured by creating the file `DISTBASE/dist/assets/
+app.config.json`. "DISTBASE" is where the Zonemaster Web GUI zip file is
```

Comment:
`DISTBASE/dist/assets/app.config.json` should be on the same line or else it is rendered with a space:
```
DISTBASE/dist/assets/ app.config.json
```

Reply:
fixed